_docs/guides/endpoints.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Install Istio for Google Cloud Endpoints Services
+overview: Guideline to manually integrate Google Cloud Endpoints services with Istio.
+
+order: 42
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This document provides a guideline to manually integrate Istio with existing
+Google Cloud Endpoints services.
+
+## Before you begin
+
+You need to follow the instruction to setup an Endpoints service on GKE [instructions](https://cloud.google.com/endpoints/docs/openapi/get-started-kubernetes-engine).
```

Comment:
The intro says "integrate Istio with existing services". So I don't think we want instructions explaining how to install an endpoint service.

Reply:
added clarifications